image/image_client.py

```diff
@@ -0,0 +1,11 @@
+import hashlib
+import requests
+from io import BytesIO
+
+def get(url):
+    response = requests.get(url)
+    return BytesIO(response.content)
+
+def hexdigest(image_binary):
```

Comment:
this is the nittiest of nits, but I don't think this fits with a client. OTOH, it seems silly to have two image related modules with a single method each